Source/DotSpatial.Analysis/VectorToRaster.cs

```diff
@@ -128,7 +128,8 @@ public static IRaster ToRaster(IFeatureSet fs, Extent extent, double cellSize, s
                         PolygonScheme ps = new PolygonScheme();
                         colorTable = ps.GenerateUniqueColors(fs, fieldName);
                         mpl.Symbology = ps;
-                        mpl.DrawRegions(args, new List<Extent> { env });
+                        mpl.DrawRegions(args, new List<Extent> { env }, false);
+                        mpl.DrawRegions(args, new List<Extent> { env }, true);

```

Comment:
Ok, can we at least write comments in code about it? Here, and in all other places. This piece of code looks not obvious, and in feature questions might appear.